cluster/charts/agent/templates/deployment.yaml

```diff
@@ -0,0 +1,70 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: crossplane-agent
+  namespace: {{ .Release.Namespace }}
+spec:
+  selector:
+    matchLabels:
+      app: crossplane-agent
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: crossplane-agent
+    spec:
+      serviceAccountName: crossplane-agent
+      imagePullSecrets: {{ .Values.imagePullSecrets }}
+      containers:
```

Comment:
Should local and remote containers both always be running?

Reply:
Yeah, they watch different clusters with separate managers and set of controllers. I wanted to make it one container but it's not really possible to make a controller watch a cluster without a manager and you can have only 1 manager per binary.